### ISSUE TITLE
listview in sample app has no border reveal

### ISSUE THREAD

**Issue by touseefbsb** [Microsoft/fluent-xaml-theme-editor#25]
in the sample app when I bring the pointer close to the listview it should show the reveal highlight on the borders, but it does not, however when I hover on the listviewitems, then I can see the reveal properly. However the issue does not exist when the theme is actually exported into a uwp app.

**Comment by kikisaints** [comment]
Hmm, it might be the version of the app or your build - did you recently update, or is this something that you’ve noticed for a while?

**Comment by touseefbsb** [comment]
just noticed it today, and I have the store version installed.

**Comment by kikisaints** [comment]
Oh! I think I know what you're experiencing. This is actually intended.

In the latest release (which you may have gotten recently) we changed listview to have a slightly different Reveal behavior than say, CommandBar does. Listview will now only show reveal when hovering over the items themselves, and will no longer have an "approach" behavior (seeing Reveal borders when near the list items).

We changed Reveal for lists because we were seeing 2px thick lines between items in a list (top, bottom) but the sides were 1px. Since Reveal was mostly 1px throughout the other controls, we decided to make sure that listview followed the 1px border rule rather than display mis-matched thicknesses. Thus we turned off the approach behavior.

NavigationView kept the thicker borders between items because of it's collapsed mode looking like CommandBar and we wanted to keep the visual consistency there - although that control is still an odd one 😄

**Comment by touseefbsb** [comment]
by changed the controls do u mean changed them in the sdk? and gridview still has the approach reveal borders, which makes it different from listview. I am sure you considered this possibility but if you put 2px vertical margins between list view items the 2px reveal border problem might go away, even 1 px margin between 2 item might work as well, I think this change was made in commandbar as well in april 2018 update and now all appabarbuttons have a 1px spacing between then by default.

**Comment by touseefbsb** [comment]
I see the problem. May I suggest an optional boolean property to turn on this approach reveal on listview? ( which will be off by default to avoid the problem) and only those developer can turn it on who do not encounter the parallel listview problem in their app, because it is a very fluent and nice to have feature.
something like : '<ListView  ApproachBorderReveal="true"/>'

**Comment by kikisaints** [comment]
Definitely! I can see the usefulness for that - would you mind going over to the [WinUI Repo Issues](https://github.com/Microsoft/microsoft-ui-xaml/issues) page and logging this API suggestion as a feature request? Feel free to also link back to this issue in the comments for extra context.

**Comment by touseefbsb** [comment]
Done :)
